lib/guest_installation_and_configuration_base.pm

```diff
@@ -1699,7 +1699,7 @@ sub post_fail_hook {
     $self->reveal_myself;
     $self->upload_guest_installation_logs;
     save_screenshot;
-    virt_utils::collect_host_and_guest_logs("", "", "/root");
+    virt_utils::collect_host_and_guest_logs("", "", "/root /var/log");
```

Comment:
May I ask what specific logs are useful but not included in supportconfig according to your observation? This directory has massive logs and some duplication with supportconfig. Just uploading the whole directory is not a good idea I am afraid, and also waste of disk space.

Reply:
This collection only happens on installation failure, so it is not massive. This folder does not contain all the stuff that you think might be there. It only contains installation related logs, for example zypper history/log, mkinitrd log and etc. And it does not only server sles guest, but also other linux os. So this is necessary and absolutely not waste of disk space. I already provided verification run and hope you can have a detailed look.